cpp/oneapi/dal/backend/primitives/stat/BUILD

```diff
@@ -0,0 +1,28 @@
+package(default_visibility = ["//visibility:public"])
+load("@onedal//dev/bazel:dal.bzl",
+    "dal_module",
+    "dal_test_suite",
+)
+
+dal_module(
+    name = "stat",
+    auto = True,
+    dal_deps = [
+        "@onedal//cpp/oneapi/dal:table",
+        "@onedal//cpp/oneapi/dal/backend/primitives:common",
+        "@onedal//cpp/oneapi/dal/backend/primitives:blas",
+    ],
+)
+
+dal_test_suite(
+    name = "tests",
+    framework = "catch2",
+    compile_as = [ "dpc++" ],
+    srcs = glob([
+        "test/*_dpc.cpp",
+    ]),
+    dal_deps = [
+        ":stat",
+    ],
+    tags = [ "gpu-only" ],
```

Comment:
Does it mean that only GPU primitives exist - no CPU version?

Reply:
I'll remove that